ts/README.md

```diff
@@ -155,6 +155,12 @@ of them:
 * Spread objects into objects:
   `const obj = { cool: true }; const clone = { isClone: true, ...obj };`
 
+* Optional chaining for props that might not exist:
+  `const option = grandparent?.parent?.myOption;`
+
+* Nullish coalescing operator instead of pick():
+  `const option = optionThatCanBeNull ?? defaultOption;`
```

Comment:
Maybe you should change the `optionThatCanBeNull ` to `canBeNullOrUndefined`? It could make the whole pattern clear for everyone imho.